docs/pages/technical-specs/expo-updates-1.mdx

```diff
@@ -185,6 +202,23 @@ type ExpoAssetHeaderDictionary = {
 
 - `assetRequestHeaders`: MAY contain a dictionary of header (key, value) pairs to include with asset requests. Key and value MUST both be strings.
 
+### Directive Body
+
+Defined as JSON conforming to both the following `Directive` definition expressed in [TypeScript](https://www.typescriptlang.org/) and the detailed descriptions for each field:
+
+```typescript
+type Directive = {
+  directiveType: string;
+  directivePayload?: { [key: string]: any };
+  [key: string]: any;
+};
+```
+
+- `directiveType`: The type of directive directive.
+- `directivePayload`: MAY contain any extra information specific to the `directiveType`.
+
+A conformant client library and server SHOULD specify and implement directive types specific to the needs of the application. For example, Expo Application Services makes use of two directive types: `noUpdateAvailable` (directs the expo-updates library to proceed as normal) and `rollbackToEmbedded` (directs the expo-updates library to use the update embedded in the host application instead of any other downloaded updates).
```

Comment:
Since all three multipart parts are optional, what if instead of a `noUpdateAvailable` directive, we had the spec treat an empty multipart response as a no-op? That would also save on response bytes. The client should also accept either a 204 or 200 in that case.